com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/MatrixTransposeNode.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class MatrixTransposeNode : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1,
+            "MatrixTranspose",
+            "Out = transpose(In);",
+            new ParameterDescriptor("In", TYPE.Matrix, Usage.In, new float[] { 1f, 0f, 0f, 1f}),
```

Comment:
What should the default size for matrices be?

This is fine for now, but we'll probably want to hook up defaulting to identity at a lower level.

Reply:
In the old system, the default is 2x2 identity, so this is following that pattern.